storage/backup/backup-service/src/lib.rs

```diff
@@ -12,7 +12,7 @@ use std::{
 use tokio::runtime::{Builder, Runtime};
 
 pub fn start_backup_service(port: u16, db: Arc<LibraDB>) -> Runtime {
-    let address = SocketAddr::new(IpAddr::V4(Ipv4Addr::LOCALHOST), port);
+    let address = SocketAddr::new(IpAddr::V4(Ipv4Addr::UNSPECIFIED), port);
```

Comment:
What does this line do?

Reply:
LOCALHOST is 127.0.0.1
UNSPECIFIED is 0.0.0.0